### ISSUE TITLE
Axillary nerve distribution

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11071]
**Preferred term label:**
Axillary nerve distribution

**Synonyms**

**Definition (free text, please give PubMed ID)**
Paresthesias on the lateral aspect of the shoulder and upper arm. Weakness in shoulder abduction (deltoid) and shoulder external rotation (teres minor).

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Quadrilateral space syndrome
Proximal humerus fracture
Shoulder dislocation

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001270
- label: Axillary nerve distribution
- parents: [Spatial pattern (HP:0012836)](https://hpo.jax.org/app/browse/term/HP:0012836)
- pmid: 
- def: Paresthesias on the lateral aspect of the shoulder and upper arm. Weakness in shoulder abduction (deltoid) and shoulder external rotation (teres minor).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11071
- synonyms:
